src/Entity/MajorVersion.php

```diff
@@ -132,6 +132,13 @@ public function getReleases(): Collection
         return $this->releases;
     }
 
+    public function removeEltsReleases(): void
+    {
+        $this->releases = $this->releases->filter(static function (Release $release) {
```

Comment:
please don't do this in the entity, do it in the repository and add a dedicated method for it.